ui/cypress/integration/integration/settings_spec.js

```diff
@@ -0,0 +1,16 @@
+describe("Settings", () => {
+    describe("Service Meshe", () => {
```

Comment:
@piyushsingariya, we might be missing an "s" here.

Reply:
Yeah, a Typo.